proto/milkyway/investors/v1/messages.proto

```diff
@@ -11,12 +11,35 @@ option go_package = "github.com/milkyway-labs/milkyway/v10/x/investors/types";
 // Msg defines the investors module's gRPC message service.
 service Msg {
   option (cosmos.msg.v1.service) = true;
+
+  // SetVestingInvestor defines a (governance) operation for adding a new
+  // vesting investor. The authority defaults to the x/gov module account.
+  rpc SetVestingInvestor(MsgSetVestingInvestor) returns (MsgSetVestingInvestorResponse);
```

Comment:
Can you please name it `AddVestingInvestor` since it's a better name than `SetVestingInvestor`?